spec/lib/evil_query_spec.rb

```diff
@@ -66,10 +66,10 @@
 
   describe "ordering" do
     before do
-      @status_messageA = FactoryGirl.create(:status_message, :author => bob.person)
-      @status_messageB = FactoryGirl.create(:status_message, :author => bob.person)
-      @status_messageD = FactoryGirl.create(:status_message, :author => bob.person)
-      @status_messageE = FactoryGirl.create(:status_message, :author => bob.person)
+      @status_messageA = FactoryBot.create(:status_message, :author => bob.person)
+      @status_messageB = FactoryBot.create(:status_message, :author => bob.person)
```

Comment:
- Naming/VariableName: Use snake_case for variable names.
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.